src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div>
+      <form
+      action=""
+      data-qa="big"
+      >
```

Comment:
```suggestion
      <form
        action="#"
        data-qa="big"
      >
```
Не оставляй action пустым, и не забывай про отступы у атрибутов